packages/material-ui/src/Select/Select.js

```diff
@@ -200,12 +203,10 @@ Select.propTypes = {
   onOpen: PropTypes.func,
   /**
    * Control `select` open state.
-   * You can only use it when the `native` prop is `false` (default).
```

Comment:
While you certainly can use it you won't see any effect. I'd argue it's highly misleading since  I would expect `<select open />` to have the options visible. But that's not the case. You can't control the open state of native `<select>`.

We should probably add a warning as well but can do that in another PR. Though I don't understand why we would completely remove the note.

Reply:
Oh agree, I thought it was saying "also", my bad 🤦‍♂️

Reply:
Does this also apply to `renderValue`? I think that doesn't work for native as well.

Reply:
Correct, we didn't implement it, yet: https://github.com/mui-org/material-ui/issues/22074.

Reply:
Should I revert these lines?

Reply:
Yes please